### PR TITLE
New Storage System

### PR DESCRIPTION
**Pull request by SirFaizdat**
This is a new storage system for Prison. It allows for multiple storage backends and adding more dat without hassle. The SQL and MongoDB storage classes have not yet been written; however, the flatfile works so it is ready for merge (after review, of course).

Related issue: #4

### CODE REVIEW

File: .idea/modules/prison-core/prison-core.iml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (DMP9-MC) [inline .idea/modules/prison-core/prison-core.iml]: IDE files should be excluded

**Comment by SirFaizdat** [comment]
Thanks, I'm working on some bug-fixes and refinements to the system right now.

**Comment by DMP9-MC** [comment]
Cool. Sorry if i messed idea when you pulled XD

**Comment by DMP9-MC** [comment]
The release date is rapidly approaching. How much (do you think) needs to be done to this PR before it can be merged?

**Comment by SirFaizdat** [comment]
I hope to have it done for today.

**Comment by SirFaizdat** [comment]
Merged manually

**Pull request closed without merging**